Northstar.Client/mod/scripts/vscripts/ui/menu_mode_select.nut

```diff
@@ -22,6 +25,20 @@ void function InitModesMenu()
 	AddMenuFooterOption( menu, BUTTON_SHOULDER_RIGHT, "#PRIVATE_MATCH_PAGE_NEXT", "#PRIVATE_MATCH_PAGE_NEXT", CycleModesForward, IsNorthstarServer )
 }
 
+bool function RegisterPlaylistBannerImage(string name, asset image)
```

Comment:
I'm not sure if its more efficient but you could probably change `image` to `string` and use `StringToAsset` instead.

No clue whether it helps, probably just needless worry.

Reply:
I don't think it would really change anything, and I would assume most people just copy whatever is returned when printing the image of the banner, which is formatted as an asset. Also, there's not really much point passing it as another type only to immediately change it.

Reply:
I think the idea is to have the path itself as the playlistvar and then just turning that into an asset with StringToAsset

Reply:
Oh I see, yeah that would work

Reply:
Though that may cause issues with gamemode icons which I realise I need to push, since game mode icons and images use the same var, could just add another var for icon overrides though